setup.py

```diff
@@ -23,6 +23,9 @@
     ],
     packages=find_packages(),
     include_package_data=True,
+    package_data={'jwst_reffiles': ['bad_pixel_masks/*.txt',
+                                    'tests/test_data/file_for_completed_steps_test.fits']
```

Comment:
Probably don't need to include test data in the packaged sdist, as the tests themselves are not included.  They live outside the jwst_reffiles directory.

If the runtime code, such as bad pixel mask generator does depend on this test FITS file, then it might be good to move it to the actual package.